frontend/src/pages/Home.jsx

```diff
@@ -1,9 +1,15 @@
 import App from "../App";
+import RoundedBox from "../components/UI/RoundedBox";
 
 const Home = () => {
     return (
         <App>
-            <h1 style={{ textAlign: "center" }}>This is the OPRE OPS system prototype.</h1>
+            <div className="display-flex flex-justify-center">
+                <RoundedBox className="padding-x-2 margin-top-2 display-inline-block text-center">
+                    <h1>This is the OPRE OPS system prototype</h1>
```

Comment:
Is prototype the appropriate word at this point?    I'm not sure what I would suggest as an alternative but something to stop and consider I feel like at least. @kimschulke @tdonaworth @fpigeonjr @johndeange @stevtek @xlorepdarkhelm

Reply:
`Beta / WIP` ?